plugins/pastetools/filter/image.js

```diff
@@ -97,8 +252,77 @@
 		return ret;
 	}
 
+	function getImageType( imageContent ) {
+		var tests = [
```

Comment:
Maybe we could extract it the same way as `CKEDITOR.pasteFilters.image.supportedImageTypes`? Probably could be named something like `recognizableImageTypes`?